### PR TITLE
Simplify utils code, leveraging golang's libs where appropriate

### PR DESCRIPTION
**Pull request by ericbottard**

### CODE REVIEW

File: riff-cli/cmd/common_options.go [hunk 1]
```diff
@@ -25,16 +25,6 @@ import (
 	"github.com/projectriff/riff/riff-cli/pkg/osutils"
 )
 
-func validateFilepath(path *string) error {
-	*path = filepath.Clean(*path)
-
-	if *path == "" {
```
Comment (ericbottard) [inline riff-cli/cmd/common_options.go]: `Clean` already does that for us

File: riff-cli/pkg/functions/functions.go [hunk 2]
```diff
@@ -33,27 +34,3 @@ func FunctionNameFromPath(path string) (string, error) {
 	return filepath.Base(filepath.Dir(abs)), nil
 }
 
-func FunctionDirFromPath(path string) (string, error) {
```
Comment (ericbottard) [inline riff-cli/pkg/functions/functions.go]: Was unused

File: riff-cli/pkg/functions/functions.go [hunk 3]
```diff
@@ -33,27 +34,3 @@ func FunctionNameFromPath(path string) (string, error) {
...
-	if err != nil {
-		return "", err
-	}
-	if osutils.IsDirectory(abs) {
-		return abs, nil
-	}
-	return filepath.Dir(abs), nil
-}
-
-func AbsPath(path string) (string, error) {
```
Comment (ericbottard) [inline riff-cli/pkg/functions/functions.go]: Moved to other package where it belongs more

File: riff-cli/pkg/osutils/osutils.go [hunk 4]
```diff
@@ -106,15 +121,11 @@ func ExecStdin(cmdName string, cmdArgs []string, stdin *[]byte, timeout time.Dur
 	}
 	// This time we can simply use Output() to get the result.
 	out, err := cmd.Output()
-	if err != nil {
```
Comment (ericbottard) [inline riff-cli/pkg/osutils/osutils.go]: These bubble up and are now correctly handled by caller

**Comment by glyn** [comment]
LGTM. merged

**Pull request merged**